dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesCommandParser.java

```diff
@@ -0,0 +1,155 @@
+package io.dockstore.client.cli.nested;
+
+import java.util.List;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+
+public class WesCommandParser {
+
+    public WesMain wesMain;
+    public CommandLaunch commandLaunch;
+    public CommandCancel commandCancel;
+    public CommandStatus commandStatus;
+    public CommandServiceInfo commandServiceInfo;
+    public JCommander jCommander;
+
+    public WesCommandParser() {
+        this.wesMain = new WesMain();
+        this.commandLaunch = new CommandLaunch();
+        this.commandCancel = new CommandCancel();
+        this.commandStatus = new CommandStatus();
+        this.commandServiceInfo = new CommandServiceInfo();
+
+        this.jCommander = buildWesCommandParser();
+    }
+
+    private JCommander buildWesCommandParser() {
+
+        // Build JCommander
+        return JCommander.newBuilder()
+            .addObject(this.wesMain)
+            .addCommand("launch", this.commandLaunch)
+            .addCommand("cancel", this.commandCancel)
+            .addCommand("status", this.commandStatus)
+            .addCommand("service-info", this.commandServiceInfo)
+            .build();
+    }
+
+    @Parameters(separators = "=", commandDescription = "Execute WES commands")
+    public static class WesMain {
+        @Parameter(names = "--wes-url", description = "The URL of the WES server.", required = false)
+        private String wesUrl = null;
+        @Parameter(names = "--wes-auth",
+            description = "The authorization type and value of this wes request. --wes-auth <type> <value>. "
+                + "Type can be 'bearer' or 'aws'. "
+                + "Value can be a token if the type is 'bearer', or an AWS profile if the type is 'aws'",
+            variableArity = true,
+            required = false)
+        private List<String> wesAuth = null;
+        @Parameter(names = "--aws-config", description = "A path to an AWS configuration file containing AWS profile credentials.", required = false)
+        private String awsConfig = null;
+        @Parameter(names = "--aws-region", description = "The AWS region of the WES server.", required = false)
```

Comment:
1. I'm not convinced the Dockstore CLI should have this argument. I'd say force the user to put the region in the AWS profile.
2. If we keep it, `... of the WES server` part of the message should not be generic like that; it's only going to apply to AGC, and the message should reflect that.

I haven't looked at your other PR yet, so we can carry this on there, and I'll approve.